dame_flame/grouped_mr.py

```diff
@@ -81,7 +80,7 @@ def algo2_GroupedMR(df_all, df_unmatched, covs_match_on, all_covs, treatment_col
                     group_covs.append('*')
             # add that group to the newly matched units to our new dataframe
 
-            return_groups.loc[newly_matched,:] = group_covs
+            return_groups.loc[newly_matched, :] = group_covs
 
             # OTHER IDEA:
             # store the bi in a column with df_all and also a column for "pair" with another
```

Comment:
This whole file is great! Just make sure you sync changes, I did have to make a minor update per Jesse's suggestions, then I'll merge your changes to this file as they are.

Reply:
I'm excited to get this one in! Looks really nice and polished with the improvements